### ISSUE TITLE
Username limitation

### ISSUE THREAD

**Issue by Afr0** [Afr0/Project-Dollhouse#116]
Make it impossible (I.E send error to client) to create accounts with names containing "username", only numbers or obscenities.

**Comment by RHY3756547** [comment]
@LetsRaceBwoi is working on this, See pull request #147.

**Comment by RHY3756547** [comment]
Just reread this- we should also allow usernames with numbers. ;) ;) ;) ;) Probably should only block Sim Names with numbers.

**Comment by LetsRaceBwoi** [comment]
Yeah but if it's only numbers (i.e. "9999999") then you don't really want those.

**Comment by Afr0** [comment]
Ok, I'll revert it :)
